test/ejabberd_tests/tests/muc_light_SUITE.erl

```diff
@@ -722,6 +724,19 @@ edge_case_owner_change(Config) ->
             escalus:assert(is_iq_result, escalus:wait_for_stanza(Alice))
         end).
 
+adding_wrongly_named_user_triggers_infinite_loop(Config)->
+    escalus:story(Config, [{alice, 1}], fun(Alice) ->
+            BuggyRoomName = <<"buggyroom">>,
+            Username = <<"buggyuser">>,
+            escalus:send(Alice, generate_buggy_aff_staza(BuggyRoomName, Username)),
+            timer:sleep(300),
+            [SessionRecPid] = rpc(ets, tab2list, [session]),
+            {session, {_, Pid}, _, _, _, _} = SessionRecPid,
+            %% maybe throws exception
+            check_process_memory_growing(Pid, 0, 2),
```

Comment:
I think it should be called `assert_process_memory_not_growing`, because right now it looks like the test expects the process to grow.